packages/asset-swapper/src/utils/alt_mm_implementation_utils.ts

```diff
@@ -211,13 +215,40 @@ export async function returnQuoteFromAltMMAsync<ResponseT>(
         }
     }
 
-    const response = await axiosInstance.post(`${url}/quotes`, data, {
-        headers: { Authorization: `Bearer ${apiKey}` },
-        timeout: maxResponseTimeMs,
-    });
+    // let response: AxiosResponse<any>;
+    const response = await axiosInstance
+        .post(`${url}/quotes`, data, {
+            headers: { Authorization: `Bearer ${apiKey}` },
+            timeout: maxResponseTimeMs,
+        })
+        .catch(err => {
+            warningLogger(err, `Alt RFQ MM request failed`);
+            throw new Error(`Alt RFQ MM request failed`);
+        });
+
+    // empty response will get filtered out in validation
+    const emptyResponse = {};
 
+    // tslint:disable-next-line:custom-no-magic-numbers
+    if (response.status !== SUCCESS_CODE) {
+        const rejectedRequestInfo = {
+            status: response.status,
+            message: response.data,
+        };
+        warningLogger(rejectedRequestInfo, `Alt RFQ MM did not return a status of ${SUCCESS_CODE}`);
+        return {
+            data: (emptyResponse as unknown) as ResponseT,
```

Comment:
Why did you define `emptyResponse` here? can't you just do `data: {}`? both here and on L247?

Reply:
I believe this is to appease typescript

Reply:
no type assertion on object literals